src/FunderMaps.WebApi/Controllers/Report/InquiryController.cs

```diff
@@ -216,7 +216,10 @@ public async Task<IActionResult> SetStatusReviewAsync(int id, StatusChangeDto in
 
             // Act.
             await _inquiryRepository.SetAuditStatusAsync(inquiry.Id, inquiry);
-            await _notificationService.NotifyByEmailAsync(new string[] { "info@example.com" }, input.Message);
+            await _notificationService.NotifyByEmailAsync(
+                address: new string[] { "info@example.com" }, // TODO:
+                content: input.Message,
+                subject: "FunderMaps - Rapportage ter review");
```

Comment:
Place in constants, don't hardcode

Reply:
Not in a constant, but in the resource table.

Reply:
See #353